server/game/cards/08.6-SAT/GourdinDeBoisFlotte.js

```diff
@@ -0,0 +1,33 @@
+const DrawCard = require('../../drawcard.js');
+
+class GourdinDeBoisFlotte extends DrawCard {
+    setupCardAbilities(ability) {
+        this.attachmentRestriction({ trait: 'Drowned God' });
+
+        this.whileAttached({
+            effect: ability.effects.dynamicStrength(() => this.controller.deadPile.length)
+        });
+
+        this.interrupt({
+            when: {
+                onCharacterKilled: event => event.cardStateWhenKilled === this.parent
+            },
+            target: {
+                cardCondition: card =>
+                    card.location === 'play area' && card.getType() === 'character'
+                    && this.controller.canAttach(this, card)
```

Comment:
This needs an extra `card !== this.parent` check so the ability does not prompt in case there are no other legal parents.